tests/handlers/test_oidc.py

```diff
@@ -167,7 +181,7 @@ def reset_mocks(self) -> None:
         self.render_error.reset_mock()
         self.complete_sso_login.reset_mock()
 
-    def metadata_edit(self, values):
+    def metadata_edit(self, values: dict) -> _patch[Mock]:
```

Comment:
Possibly `typing.ContextManager[Mock]` here? (If only to avoid importing a `_private` symbol)